content/docs/studio/index.md

```diff
@@ -9,11 +9,8 @@ experiments, and visualize and share results.
 
 https://www.youtube.com/watch?v=hKf4twg832g
 
-<admon>
-
-Note that we have renamed DVC Studio to Iterative Studio and Views to Projects.
-
-</admon>
+_Note that we have renamed DVC Studio mentioned in the above video to Iterative
```

Comment:
Wondering if we should schedule removing of these messages + such messages in the Studio itself as a good amount of time has already passed since the renaming.

Reply:
Yes, I've added a note [here](https://github.com/iterative/dvc.org/issues/4092#issuecomment-1439908359).